diff --git a/Makefrag b/Makefrag
index 2b87ece9f..c6cbb7068 100644
--- a/Makefrag
+++ b/Makefrag
@@ -31,7 +31,7 @@ sbtopts_file := $(base_dir)/.sbtopts
 ifneq (,$(wildcard $(sbtopts_file)))
 	SBT_OPTS ?= $(shell cat $(sbtopts_file))
 endif
-SBT ?= java -Xmx$(JVM_MEMORY) -Xss8M -jar $(base_dir)/sbt-launch.jar
+SBT ?= /usr/lib/jvm/java-8-openjdk-amd64/bin/java -Xmx$(JVM_MEMORY) -Xss8M -jar $(base_dir)/sbt-launch.jar
 SHELL := /bin/bash
 
 FIRRTL_TRANSFORMS := \
@@ -39,7 +39,7 @@ FIRRTL_TRANSFORMS := \
 
 FIRRTL_TEST_JAR ?= $(base_dir)/firrtl/utils/bin/firrtl-test.jar
 
-JAVA ?= java -Xmx$(JVM_MEMORY) -Xss8M
+JAVA ?= /usr/lib/jvm/java-8-openjdk-amd64/bin/java -Xmx$(JVM_MEMORY) -Xss8M
 FIRRTL ?= $(JAVA) -cp $(ROCKET_CHIP_JAR) firrtl.stage.FirrtlMain
 GENERATOR ?= $(JAVA) -cp $(ROCKET_CHIP_JAR) $(PROJECT).Generator
 
